build.gradle

```diff
@@ -33,6 +33,9 @@ sourceSets {
         java {
             srcDir 'cases/src'
             srcDir 'backend/src'
+            srcDir 'api'
+            srcDir 'core'
+            srcDir 'modern'
```

Comment:
Not necessary, but would be nice to start using gradle's directory layout standards. Specifically, I think gradle's java plugin by default expects something like `src/main/...` (should double check if you actually follow up on this advice).